python/tests/kernel/test_observe_kernel.py

```diff
@@ -265,7 +265,8 @@ def circuit(theta: float):
         ry(theta, q[1])
         x.ctrl(q[1], q[0])
 
-    h = spin.z(0)
+    h = spin.empty()
```

Comment:
calling distribute_terms on a product operator will give a deprecation warning. The function only really makes sense on a sum.